.github/workflows/docs.yml

```diff
@@ -26,15 +29,21 @@ jobs:
       - uses: actions/checkout@v4
       - name: Run sccache-cache
         uses: mozilla-actions/sccache-action@v0.0.5
-      - name: Install poetry
-        run: pipx install poetry
-      - name: Set up Python '3.10'
+
+      - name: Set up uv
+        run: curl -LsSf https://astral.sh/uv/0.3.4/install.sh | sh
```

Comment:
It's a little bit scary to install via curl in our CI - I guess there's no actoin for installing uv yet?

Reply:
It's the recommended method for CI runs :/
https://docs.astral.sh/uv/guides/integration/github/#installation

But yeah, I hope they create an action soon.